src/styles/colors.ts

```diff
@@ -10,13 +10,13 @@ export default {
 	pink: '#ED1EFF',
 	green: '#31D8A4',
 	gradients: {
-		darkBlue: 'linear-gradient(180deg, #08021E 0%, #1F0777 146.21%);',
-		lightBlue: 'linear-gradient(73.6deg, #85FFC4 2.11%, #5CC6FF 90.45%);',
-		purple: 'linear-gradient(73.6deg, #8E2DE2 2.11%, #4B01E0 90.45%);',
-		orange: 'linear-gradient(73.6deg, #FF8060 2.11%, #FEB27A 90.45%);',
-		pink: 'linear-gradient(73.6deg, #8E2DE2 2.11%, #ED1EFF 90.45%);',
-		rainbow: 'linear-gradient(90deg, #ED1EFF 0%, #00D1FF 100%);',
-		grey: 'linear-gradient(0deg, rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.2)), linear-gradient(311.52deg, #3D464C -36.37%, #131619 62.81%);',
+		darkBlue: 'linear-gradient(180deg, #08021E 0%, #1F0777 146.21%)',
+		lightBlue: 'linear-gradient(73.6deg, #85FFC4 2.11%, #5CC6FF 90.45%)',
+		purple: 'linear-gradient(73.6deg, #8E2DE2 2.11%, #4B01E0 90.45%)',
+		orange: 'linear-gradient(73.6deg, #FF8060 2.11%, #FEB27A 90.45%)',
+		pink: 'linear-gradient(73.6deg, #8E2DE2 2.11%, #ED1EFF 90.45%)',
+		rainbow: 'linear-gradient(90deg, #ED1EFF 0%, #00D1FF 100%)',
+		grey: 'linear-gradient(0deg, rgba(0, 0, 0, 0.2), rgba(0, 0, 0, 0.2)), linear-gradient(311.52deg, #3D464C -36.37%, #131619 62.81%)',
```

Comment:
Removed the `;`'s from here and updated its usage everywhere to be more consistent with the other colors (and this allows gradients compositions!)

Reply:
thanks!